### PR TITLE
feat：Add the time type drop-down box to the Job list

### PR DESCRIPTION
**Pull request by kogel-net**
# Description

Add the time type drop-down box to the Job list

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

**Comment by doddgu** [comment]
@wzh425 PTAL

File: src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/SchedulerJobs.razor.cs [hunk 1]
```diff
@@ -111,14 +111,14 @@ public bool Visible
 
     private JobModal? _jobModal;
 
-    public List<KeyValuePair<string, JobQueryTimeTypes>> JobQueryTimeTypes { get; set; } = new();
+    //public List<KeyValuePair<string, JobQueryTimeTypes>> JobQueryTimeTypes { get; set; } = new();
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/SchedulerJobs.razor.cs:114]: remove

File: src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/SchedulerJobs.razor.cs [hunk 2]
```diff
@@ -212,11 +225,11 @@ protected override async Task OnInitializedAsync()
             await SignalRNotifyDataHandler(schedulerTaskDto);
         });
 
-        JobQueryTimeTypes = GetEnumMap<JobQueryTimeTypes>();
+        //JobQueryTimeTypes = GetEnumMap<JobQueryTimeTypes>();
```
Comment (wzh425) [inline src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/SchedulerJobs.razor.cs:228]: remove

Comment (kogel-net) [reply src/Web/Masa.Scheduler.Web.Admin/Pages/Teams/Components/SchedulerJobs.razor.cs]: updated

**Review by wzh425 — APPROVED** [review]
LGTM

**Pull request merged**